fast_barcode_scanner/ios/Classes/FastBarcodeScannerPlugin.swift

```diff
@@ -11,6 +11,8 @@ public class FastBarcodeScannerPlugin: NSObject, FlutterPlugin, FlutterStreamHan
     var picker: ImagePicker?
     var detectionsSink: FlutterEventSink?
 
+    var captureImageDict: [String: [UInt8]?] = [String: [UInt8]?]()
```

Comment:
@nogicoder see the comment that I made in #7 . I think we should consider a disk cache for the scanned images. This would be lighter on memory usage and faster for PlatformChannel communication, and perhaps easier also for eventual file upload.

Reply:
I updated the logic according to your suggestion